pages/1.13/release-notes/1.13.0-beta1/index.md

```diff
@@ -163,23 +163,23 @@ You can automatically provision Amazon ELB (NLB) using Edge-LB pool instances on
 
 - Add support for internationalization and localization (I18N and L10N - Chinese). <!--(DCOS-39557)-->
 
-    Mesosphere DC/OS 1.13 GUI has now been translated to Mandarin Chinese. Mandarin-speaking customers and users can now easily switch the GUI language in the UI and will be able to interact with DC/OS operations and functions in English or Chinese. The DC/OS documentation has also been translated to Chinese to support those customers. Support for additional languages can be provided if there's sufficient customer demand.
+    Mesosphere DC/OS 1.13 GUI has now been translated into Chinese. Customers and users can now easily switch the GUI language in the UI to interact with DC/OS operations and functions in English or Chinese. The DC/OS documentation has also been translated to Chinese to support those customers. Support for additional languages can be provided if there's sufficient customer demand.
```

Comment:
Written Chinese is not tied to any particular pronunciation, or any particular version of the spoken language. These documents can be read by speakers of Mandarin, Cantonese, Shanghainese languages, etc.